<launch>
    <node name="viservo_control_executor" pkg="hdt" type="viservo_control_executor" output="screen">
        <rosparam command="load" file="$(find rcta)/config/attached_markers.yaml"/>
        <rosparam command="load" file="$(find rcta)/config/viservo_control_executor.yaml"/>
        <remap from="ar_pose_marker" to="attached_ar_pose_marker"/>
    </node>
</launch>
